phpunit.xml.dist

```diff
@@ -21,6 +21,8 @@
   </coverage>
   <php>
     <ini name="error_reporting" value="-1"/>
+    <env name="AWS_SHARED_CREDENTIALS_FILE" value="/dev/null" />
```

Comment:
Not related to that PR but tests are broken if your machine has a region configured in your Default AWS profile

Reply:
That is correct. 

It is also the correct behavior. 

Another fix for this would be to set `AWS_REGION=us-east-1`.